src/menu.ts

```diff
@@ -93,6 +94,14 @@ function getAppMenu(
 						sendIpcEventToRenderer( 'user-settings' );
 					},
 				},
+				...( process.platform === 'darwin' && process.env.NODE_ENV === 'development'
+					? [
+							{
+								label: __( 'Install CLI…' ),
```

Comment:
Are we going to have this button in prod when we ungate feature?
I thought that installing CLI will be done automattically, not manually. Maybe I missed this info somewhere, could you please explain?

So if this is tmp dev button, then we should avoid translation request.
```suggestion
								label: 'Install CLI…',
```

Reply:
We can install the CLI automatically on Windows by modifying the `PATH` environment variable. However, not on macOS because we can't modify the `PATH` environment variable (persistently and reliably), and admin privileges are usually needed to write to directories that are already included in `$PATH`.

Reply:
So installation on MacOS will be only manual, with clicking on "Install CLI..." button? We won't make attempt to install it on running Studio and showing popup with providing sudo access?

Reply:
That's correct. We'll announce the CLI in the What's New modal and include instructions for installing it, but we won't prompt users automatically. 

This decision was made, in part, because the CLI will have a limited feature set in this first iteration

Reply:
> because the CLI will have a limited feature set in this first iteration

Oh yeah, this point makes definitely more sense to not disturb users with early versions of the feature 👍
Thanks for the information.